kbase-extension/static/kbase/js/common/cellComponents/cellControlPanel.js

```diff
@@ -1,8 +1,4 @@
-define(['common/html', 'common/cellComponents/actionButtons', 'common/jobs'], (
-    html,
-    ActionButton,
-    Jobs
-) => {
+define(['common/html', 'common/cellComponents/actionButtons'], (html, ActionButton) => {
```

Comment:
Made this component more generic by removing the jobs-specific wording and capabilities.